packages/elements/src/components/ino-card/ino-card.scss

```diff
@@ -16,7 +16,7 @@ ino-card {
   --card-background-color: var(
     --ino-card-background-color,
     --ino-global-primary-color,
-    #{theme.color(primary, base)}
+    #{theme.color(primary, $tone: 'base')}
```

Comment:
I think you can just write `theme.color(primary, base)`, can't you?

Reply:
I think so, yes. I only changed it because my IDE was complaining for some reason...